docs/developer-docs/latest/plugins/upload.md

```diff
@@ -278,11 +282,10 @@ Adding a file attribute to a model (or the model of another plugin) is like addi
 
 In the first example below, you will be able to upload and attach one file to the avatar attribute.
 
-**Path —** `User.settings.json`.
+**Path —** `schema.json`.
```

Comment:
Suggestion:

Now we're including the full path as a comment (followed by a blank line) in the code block, like you can see here:
![Screenshot 2021-12-03 at 16 36 24](https://user-images.githubusercontent.com/4233866/144629950-1bb3aec2-d99e-4553-90b3-4f80e4f22c92.png)

So if you could do this with all code examples, that would be awesome!